recording/src/nextcloud/talk/recording/BackendNotifier.py

```diff
@@ -59,16 +61,26 @@ def doRequest(backend, request, retries=3):
     :param backend: the backend to send the request to.
     :param request: the request to send.
     :param retries: the number of times to retry in case of failure.
+    :param library: the library to use for the request. Possible values are: 'urllib', 'requests'.
```

Comment:
`library` seems to be a leftover from an older implementation that can be removed, as it is not explicitly given and the request type is detected via `isinstance`.